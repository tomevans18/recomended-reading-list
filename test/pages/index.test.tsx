import React from 'react';
import Home from '@pages/index';
import { render } from '@test/test-util';

describe('Home page', () => {
  it('matches snapshot', () => {
    const { container } = render(<Home />, {});
    expect(container).toMatchSnapshot();
  });
});
