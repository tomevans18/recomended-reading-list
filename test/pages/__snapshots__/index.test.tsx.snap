// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<div>
  <main>
    <h1
      class="title"
    >
      Welcome to 
      <a
        href="https://nextjs.org"
      >
        Next.js!
      </a>
    </h1>
  </main>
</div>
`;
